deploy/crds/kieapp.crd.yaml

```diff
@@ -1291,21 +1291,282 @@ spec:
+                                      anyOf:
+                                      - type: integer
+                                      - type: string
+                                      description: Specifies the output format of
+                                        the exposed resources, defaults to "1"
+                                      pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                                      x-kubernetes-int-or-string: true
+                                    resource:
+                                      description: 'Required: resource to select'
+                                      type: string
+                                  required:
+                                  - resource
+                                  type: object
+                                secretKeyRef:
+                                  description: Selects a key of a secret in the pod's
+                                    namespace
+                                  properties:
+                                    key:
+                                      description: The key of the secret to select
+                                        from.  Must be a valid secret key.
+                                      type: string
+                                    name:
+                                      description: 'Name of the referent. More info:
+                                        https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                        TODO: Add other useful fields. apiVersion,
+                                        kind, uid?'
+                                      type: string
+                                    optional:
+                                      description: Specify whether the Secret or its
+                                        key must be defined
+                                      type: boolean
+                                  required:
+                                  - key
+                                  type: object
+                              type: object
+                          required:
+                          - name
+                          type: object
+                        type: array
+                      extraClassPath:
+                        description: ExtraClassPath Allows to add extra jars to the
+                          application classpath separated by colon. Needs to be mounted
+                          on the image before.
+                        type: string
                       image:
-                        description: The image to use for Process Instance Migration
-                          e.g. rhpam-process-migration-rhel8, this param is optional
-                          for custom image.
+                        description: Image The image to use e.g. rhpam-<app>-rhel8,
```

Comment:
```suggestion
                        description: The image to use for Process Instance Migration e.g. rhpam-<app>-rhel8,
```

Reply:
it is default inherited from kieAppObject, it is the same for all components.